src/local/BLELocalDevice.cpp

```diff
@@ -240,9 +199,9 @@ int BLELocalDevice::scanForAddress(String address, bool withDuplicates)
   return GAP.scanForAddress(address, withDuplicates);
 }
 
-void BLELocalDevice::stopScan()
+int BLELocalDevice::stopScan()
```

Comment:
This was added just to improve robustness because we did not have error management in the stopScan call. It could be useful at application level to manage it correctly